.github/workflows/docs.yml

```diff
@@ -1,8 +1,8 @@
 name: Deploy Hugo Docs
 
 on:
-  push:
-    tags: [ '*' ]
+  release:
+    types: [published]
```

Comment:
Do we publish releases for back patched versions? If so, I think this would trigger a build when that happens and the source checked out would default to the ref that generated the release event.

Reply:
I believe "release published" is not the right trigger.
Releasing the fix for an older release would update the site with an old contents. It is not something we want.

We should rather add a branch like `website` which we could update during the release if releasing from the master branch.